ResourceSharing/WebApplication2/Controllers/ValuesController.cs

```diff
@@ -1,47 +1,46 @@
-﻿using System;
```

Comment:
Please delete this file, as it is no longer needed

Reply:
will do

Reply:
I did not find the file in the current branch, it is from another